trade-finance-manager-ui/server/helpers/user.js

```diff
@@ -5,3 +5,12 @@ export const userFullName = (user) => {
 
 export const userIsInTeam = (user, teamList) =>
   teamList.some((teamId) => user.teams.includes(teamId));
+
+
+export const isAssignedToUser = (assignedToUserId, userId) => {
```

Comment:
can be shortened to `export const isAssignedToUser = (assignedToUserId, userId) => assignedToUserId === userId`
Do we need a helper for this? Why not just use `assignedToUserId === userId` instead of calling the helper?

Reply:
I've shortened this - we could do the check without the function, but I think it's clearer to have a helper and can be used in other places? Happy to change though